ui/pages/manage_connected_accounts.tsx

```diff
@@ -0,0 +1,280 @@
+    return <Spinner />;
+  }
+
+  return (
+    <PageLayout title="Connected accounts" isSelfServe={true} user={userName}>
+      {flow && (
+        <div>
+          <p>{connectionText[connectionState]}</p>
+          {connectionState !== "none" && (
+            <>
+              <p className="p-heading--5">Manage accounts</p>
+              <List
+                items={providers.map(({ id, label, disabled, name }) => (
+                  <div key={id} className="provider">
+                    <img
+                      src={getProviderImage(label)}
+                      alt={`${label} logo`}
+                      className="provider-logo"
+                    />
+                    <span>{label}</span>
+                    {name === "link" ? (
+                      <Button
+                        className="link-provider-btn"
+                        appearance="positive"
+                        disabled={disabled}
+                        onClick={() =>
+                          handleSubmit({
+                            method: "oidc",
+                            link: id,
+                          })
+                        }
+                        hasIcon
+                      >
+                        <Icon name="get-link" />
+                        <span>Connect</span>
+                      </Button>
+                    ) : (
+                      <ConfirmationButton
+                        disabled={disabled}
+                        appearance="negative"
+                        className="unlink-provider-btn has-icon"
+                        confirmationModalProps={{
+                          title: "Disconnect Account?",
+                          confirmButtonLabel: "Disconnect",
+                          onConfirm: () =>
+                            void handleSubmit({
+                              method: "oidc",
+                              unlink: id,
+                            }),
+                          children: (
+                            <>
+                              <p className="u-no-margin--bottom">
+                                You&apos;re about to disconnect your {label}{" "}
+                                account from this profile.
+                              </p>
+                              <p className="u-no-margin--bottom">
+                                This will remove {label} as a login method.
+                                Ensure you have an alternative method to access
+                                your account.
+                              </p>
+                              <p>
+                                You may reconnect it at any time in your
+                                settings.
+                              </p>
+                            </>
+                          ),
+                        }}
+                      >
+                        <Icon name="delete" />
+                        <span>Disconnect</span>
```

Comment:
question: Can we align on the wording with the UX team? The designs use terms 'add account' and 'remove', while here we have 'connect' and 'disconnect'.

Same question for the displayed messages, like `You haven’t connected any accounts yet. Connect an account from a service as another way to sign in quickly and securely.`